src/app/public/modules/date-range-picker/date-range-picker.component.ts

```diff
@@ -178,6 +180,20 @@ export class SkyDateRangePickerComponent
   @Input()
   public label: string;
 
+  /**
+   * Indicates whether to mark the start date input as required.
```

Comment:
```suggestion
   * Indicates whether to require users to specify a start date.
```
I assume this property actually makes the property required and doesn't just visually mark it as required.

Reply:
Done.